invenio/base/templates/_formhelpers_base.html

```diff
@@ -144,3 +146,17 @@
     {% endif %}
   {% endfor %}
 {% endblock %}{% endmacro %}
+
+{% macro render_radio_field(field) -%}
+<div class="form-group">
+  <label class="control-label" for="name">{{ field.label }}</label>
+  {% for value, label, checked in field.iter_choices() %}
+  <div class="radio">
+    <label>
+      <input type="radio" name="{{ field.id }}" id="{{ field.id }}" value="{{ value }}" {{ "checked" if checked }}>{{ label }}
+      <small class="text-muted">{{ field.description[value] }}</small>
```

Comment:
@lnielsen, can you take a look at it?
Here I used a little trick to get description per bullet.
Here is how I'm using it: https://github.com/inveniosoftware/invenio-groups/pull/1/files#diff-53e9e6c85499508d2e7be4767059f1daR149
I'm not sure if this could be done any better?